src/legacy/plugin_discovery/plugin_spec/plugin_spec_options.d.ts

```diff
@@ -23,6 +23,8 @@ export type InitPluginFunction = (server: Server) => void;
 export interface UiExports {
   injectDefaultVars?: (server: Server) => { [key: string]: any };
   styleSheetPaths?: string;
+  savedObjectsManagement?: any; // Not ideal
```

Comment:
Thanks! Looks like this is actually for saved objects management (not management sections in general).

But since you've pointed it out, I think there actually is an interface that could be used here: https://github.com/elastic/kibana/blob/4918baa9eed978b8e986b7f70975754fbf059240/src/core/server/saved_objects/management/management.ts#L31-L33

(Haven't used this interface before, but pretty sure that's the right one)

Reply:
We tried that type but got a "no imports from restricted zone" linter error. We noticed a file in the same module is using the same type [with an eslint ignore comment](https://github.com/elastic/kibana/blob/4fa253ab93d2bd4dc588143040cd9660e21ba2c4/src/legacy/plugin_discovery/types.ts#L25-L26), so we decided to follow that precedent. Let us know if either of you know if that's a terrible idea.